store/sqlstore/team_store.go

```diff
@@ -1022,6 +1023,7 @@ func (s SqlTeamStore) GetMember(teamId string, userId string) (*model.TeamMember
 	return dbMember.ToModel(), nil
 }
 
+// GetMembers returns a list of members from the database that match the teamId passed as parameter
```

Comment:
I would expect some extra information here around the `teamMemberGetOoptions` parameter, something like "the teamMembersGetOptions parameter allow you to restrict what is visible to the user, sort the results or exclude deleted users".